articles/quantum-InstallConfig.md

```diff
@@ -15,131 +15,28 @@ ms.topic: article
 # Installing the Quantum Development Kit
 
 The Quantum Development Kit can be used either with the Visual Studio 2017 integrated development environment, or with the command line and a development editor such as Visual Studio Code.
-We'll walk through both methods in this install guide.
+
+<table>
+<tr>
+<td>
+[<img src="media/install-guide-select-vs-2017.png" width="90%">](xref:microsoft.quantum.install.vs-2017)
+<div class="nextstepaction">
+[Install for Visual Studio 2017 on Windows 10](xref:microsoft.quantum.install.vs-2017)
```

Comment:
Install *for*, or Install *on*?

Reply:
I didn't want to use the word "on" in both places, but I agree, not quite right yet. I'll reword accordingly, then.